src/main/com/sailthru/client/params/User.java

```diff
@@ -76,6 +78,16 @@ public User setLogin(Map<String, Object> login) {
         return this;
     }
 
+    public User setSms_marketing_status(String sms_marketing_status) {
```

Comment:
Are we able to follow standard Java naming conventions for methods being camel-case?  https://www.javatpoint.com/java-naming-conventions
```suggestion
    public User setSmsMarketingStatus(String smsMarketingStatus) {
```

Reply:
changed the method name to adhere to the java naming conventions